includes/push-ui.php

```diff
@@ -83,10 +83,6 @@ function syndicatable() {
 		return;
 	}
 
-	if ( ! in_array( $post->post_status, \Distributor\Utils\distributable_post_statuses(), true ) ) {
-		return false;
-	}
-
 	if ( ! in_array( get_post_type(), $distributable_post_types, true ) || ( ! empty( $_GET['post_type'] ) && 'dt_ext_connection' === $_GET['post_type'] ) ) { // @codingStandardsIgnoreLine Nonce not required
 		return false;
 	}
```

Comment:
Not entirely related but more readable. 

```suggestion
	$distributable_post_types = array_diff( $distributable_post_types, array( 'dt_ext_connection' ) );
	if ( ! in_array( get_post_type(), $distributable_post_types, true ) ) {
		return false;
	}
```


It's outside the diff but change `global $post` to `$post = get_post();` above this too. It's a much nicer way of getting the current global post object.